### PR TITLE
Twingate.Client version 1.0.21.14067

### PR DESCRIPTION
**Pull request by chenbishop**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/92430)

### CODE REVIEW

File: manifests/t/Twingate/Client/1.0.21.14067/Twingate.Client.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Twingate.Client
+PackageVersion: 1.0.21.14067
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://api.twingate.com/download/windows
+  InstallerSha256: F73772DC74E7D82506C7C748DBFCFB0FD92228430F2477802A557FD97E225E45
+ManifestType: installer
```
Comment (stephengillie) [inline manifests/t/Twingate/Client/1.0.21.14067/Twingate.Client.installer.yaml:11]:
```suggestion
  InstallerSwitches:
    Silent: /quiet
ManifestType: installer
```

File: manifests/t/Twingate/Client/1.0.21.14067/Twingate.Client.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Twingate.Client
+PackageVersion: 1.0.21.14067
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
```
Comment (stephengillie) [inline manifests/t/Twingate/Client/1.0.21.14067/Twingate.Client.installer.yaml:8]:
```suggestion
  InstallerType: exe
```

File: manifests/t/Twingate/Client/1.0.21.14067/Twingate.Client.installer.yaml (hunk 1 above)
Comment (stephengillie) [inline manifests/t/Twingate/Client/1.0.21.14067/Twingate.Client.installer.yaml:11]:
```suggestion
  InstallerSwitches:
    Silent: /quiet
ManifestType: installer
```

File: manifests/t/Twingate/Client/1.0.21.14067/Twingate.Client.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Twingate.Client
+PackageVersion: 1.0.21.14067
+PackageLocale: en-US
+Publisher: Twingate Labs
+PackageName: Twingate Client
```
Comment (stephengillie) [inline manifests/t/Twingate/Client/1.0.21.14067/Twingate.Client.locale.en-US.yaml:8]:
```suggestion
PublisherURL: https://www.twingate.com/
PublisherSupportUrl: https://help.twingate.com/hc/en-us
PrivacyUrl: https://www.twingate.com/privacy/
PackageName: Twingate Client
```

File: manifests/t/Twingate/Client/1.0.21.14067/Twingate.Client.installer.yaml (hunk 1 above)
Comment (stephengillie) [reply manifests/t/Twingate/Client/1.0.21.14067/Twingate.Client.installer.yaml:11]: Validation is failing due to the install switches. This is what is being encountered: 
image

I've tested the above suggestions in a VM, and together they allow for a silent install.

**Comment by chenbishop** [comment]
Hi @stephengillie, thank you very much for your help!

I have made some changes and the installer is now validated and tested (using winget validate and winget install). However,  the check is returning the following error 

Url validation error

manifests\t\Twingate\Client\1.0.24.16856
https://help.twingate.com/hc/en-us
Http status code: Forbidden


and i cannot seem to reproduce the issue using the browser. Could you give us some guidance please?

Alternatively, do you think we could remove the line below from Twingate.Client.locale.en-US.yaml to resolve the issue?

PublisherSupportUrl: https://help.twingate.com/hc/en-us

**Comment by chenbishop** [comment]
> @chenbishop please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> 
> Contributor License Agreement

@microsoft-github-policy-service agree [company="Twingate Inc."]

**Comment by chenbishop** [comment]
@microsoft-github-policy-service agree company="Twingate Inc."

**Comment by chenbishop** [comment]
> @chenbishop please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> 
> Contributor License Agreement


@microsoft-github-policy-service agree company="Twingate Inc."

**Comment by stephengillie** [comment]
> Url validation error
> 
> * manifests\t\Twingate\Client\1.0.24.16856
>   
>   * https://help.twingate.com/hc/en-us
>     
>     * Http status code: Forbidden

I can't verify this.

**Review by stephengillie — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**